Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG BASE_TAG=2019.03
 ARG TENSORFLOW_VERSION=2.1.0
 
 FROM gcr.io/kaggle-images/python-tensorflow-whl:${TENSORFLOW_VERSION}-py36-2 as tensorflow_whl
```

Comment:
we may be able to avoid the tensorflow wheel if we use the tensorflow base image (ie. gcr.io/deeplearning-platform-release/tf-cpu)